com.github.kmwallio.thiefmd.json

```diff
@@ -0,0 +1,106 @@
+{
+  "app-id":"com.github.kmwallio.thiefmd",
+  "runtime":"org.gnome.Platform",
```

Comment:
You could use the freedesktop runtime/sdk along with the elementary base app instead of the gnome runtime if all you want is vala, granite & libgee.

Reply:
The vala compiler in the elementary base app is a bit old.

If I switch to freedesktop with elementary base app, and then compile vala, does it take up less space/less resources needed for users?

Reply:
There'll be less disk space used in the case that the user only has one flatpak installed (yours). It's difficult to generalise across all scenarios but if you've tested and have specific requirements for specific libraries then using the smallest runtime is probably best. It's not a blocker though.